bgpd/bgp_vty.c

```diff
@@ -6259,12 +6260,17 @@ DEFPY (af_label_vpn_export,
 	int ret;
 	int idx = 0;
 	int yes = 1;
+	int dynamic_label = 0;
 
 	if (argv_find(argv, argc, "no", &idx))
 		yes = 0;
 
-	if (yes)
-		label = label_val; /* rely on parser to force unsigned */
+	if (yes) {
+		if (label_auto && strmatch(label_auto, "auto"))
```

Comment:
No need for the strmatch, label_auto as non-null tells you that the auto keyword was entered.